webservices/docs.py

```diff
@@ -481,8 +482,14 @@
     - U Unregistered Committee Notice
 '''
 DATES_TAG = '''
-Regular reporting deadlines and election dates.
+Reporting deadlines, election dates FEC meetings, events etc.
 '''
+
+CALANDAR_DATES = '''
```

Comment:
Thanks! I have a serious issue with spelling that right, and It apparently doesn't matter how many times I have had to type that in the last few months.